README.md

```diff
@@ -2,9 +2,25 @@
 
 Website to find a court or tribunal.
 
-## Getting Started..
-
-### Prerequisites.
+## Table of Contents
+
+* [Getting Started](#setting-started)
```

Comment:
Probably being slow, are these linking to something further down in the readme?

Reply:
I think that was just the title section for the 3 subsections below it. The bits to get started are in the subsections i think.

Reply:
ah i had a spelling mistake